<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogKeySet</class>
 <widget class="QDialog" name="DialogKeySet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="application.qrc">
    <normaloff>:/images/keyset.png</normaloff>:/images/keyset.png</iconset>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="13" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="key2_tweak_label">
     <property name="text">
      <string>BISKEY2_TWEAK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="key1_crypt_edit"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="key3_crypt_label">
     <property name="text">
      <string>BISKEY3_CRYPT</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ImportButton">
       <property name="maximumSize">
        <size>
         <width>181</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;biskeydump or lockipick format. &lt;br/&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;i.e &amp;quot;BIS KEY N (crypt): XXX...&amp;quot;  for 16-byte crypto/tweak&lt;br/&gt;or &amp;quot;bis_key_0N = XXX...&amp;quot; for 32-byte keys&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Import keys from file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="key0_tweak_edit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="key2_crypt_label">
     <property name="text">
      <string>BISKEY2_CRYPT</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="key3_tweak_label">
     <property name="text">
      <string>BISKEY3_TWEAK</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="key1_tweak_label">
     <property name="text">
      <string>BISKEY1_TWEAK</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="key2_tweak_edit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="key0_crypt_label">
     <property name="text">
      <string>BISKEY0_CRYPT</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="key3_crypt_edit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="key0_tweak_label">
     <property name="text">
      <string>BISKEY0_TWEAK</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="key1_crypt_label">
     <property name="text">
      <string>BISKEY1_CRYPT</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="key1_tweak_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="key0_crypt_edit"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="key3_tweak_edit"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="key2_crypt_edit"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_other_keys">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogKeySet</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogKeySet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
